recipes/wntr/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "wntr" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/michaelbynum/WNTR/archive/f73f0618e22201c0d15e5d1c62537564fc8840cb.zip
+  sha256: ed3b323ebe7bac1cce88520294327f61327d752f0e5086a28c909896948f799b
+
+build:
+  number: 0
+  skip: true  # [win]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - numpy
+    - python
+    - pip
+  host:
+    - python
+    - pip
+    - numpy
```

Comment:
It doesn't look like the `setup.py` imports `numpy` either directly or as a side-effect. If this is the case, you don't need to have it in the `host` section.

Reply:
@synapticarbors I suspect this comment is also related to the fact that I am working on a branch. We are using numpy's get_include method.